src/shared/modules/udc/udcDuck.js

```diff
@@ -247,6 +331,25 @@ export const bootEpic = (action$, store) => {
     })
 }
 
+const actionsOfInterest = [
+  PIN,
+  UNPIN,
+  REMOVE,
+  EXPAND,
+  ADD_FAVORITE,
+  LOAD_FAVORITES,
+  UPDATE_FAVORITE,
+  REMOVE_FAVORITE,
+  UPDATE
+]
+export const trackReduxActionsEpic = (action$, store) =>
+  action$
+    .filter(action => actionsOfInterest.includes(action.type))
```

Comment:
Might be a little easier to read as `.ofType(...actionsOfInterest)`

Reply:
Oh didn't know it took multiple arguments, neat!